contrib/endpoints/src/api_manager/auth/lib/jwt_verifier.h

```diff
@@ -26,49 +26,54 @@ extern "C" {
 #include <grpc/support/time.h>
 }
 
+namespace google {
+namespace api_manager {
+namespace auth {
+
+
 /* --- Constants. --- */
 
 #define GRPC_OPENID_CONFIG_URL_SUFFIX "/.well-known/openid-configuration"
 #define GRPC_GOOGLE_SERVICE_ACCOUNTS_EMAIL_DOMAIN "gserviceaccount.com"
 #define GRPC_GOOGLE_SERVICE_ACCOUNTS_KEY_URL_PREFIX \
   "www.googleapis.com/robot/v1/metadata/x509"
 
-/* --- grpc_jwt_verifier_status. --- */
+/* --- jwt_verifier_status. --- */
 
 typedef enum {
-  GRPC_JWT_VERIFIER_OK = 0,
-  GRPC_JWT_VERIFIER_BAD_SIGNATURE,
-  GRPC_JWT_VERIFIER_BAD_FORMAT,
-  GRPC_JWT_VERIFIER_BAD_AUDIENCE,
-  GRPC_JWT_VERIFIER_KEY_RETRIEVAL_ERROR,
-  GRPC_JWT_VERIFIER_TIME_CONSTRAINT_FAILURE,
-  GRPC_JWT_VERIFIER_BAD_SUBJECT,
-  GRPC_JWT_VERIFIER_GENERIC_ERROR
-} grpc_jwt_verifier_status;
+  JWT_VERIFIER_OK = 0,
+  JWT_VERIFIER_BAD_SIGNATURE,
+  JWT_VERIFIER_BAD_FORMAT,
+  JWT_VERIFIER_BAD_AUDIENCE,
+  JWT_VERIFIER_KEY_RETRIEVAL_ERROR,
+  JWT_VERIFIER_TIME_CONSTRAINT_FAILURE,
+  JWT_VERIFIER_BAD_SUBJECT,
+  JWT_VERIFIER_GENERIC_ERROR
+} jwt_verifier_status;
```

Comment:
coding style is very confusing now.  according to c++ style guide,  enum type should be the same as class name as camel case.  same as function names. 

I understand the code is copied from grpc,   so it did not follow the style guide and we may want to keep the same style to have minimum change.